arviz/data/base.py

```diff
@@ -1,13 +1,13 @@
 """Low level converters usually used by other functions."""
 import datetime
 import functools
+import importlib.resources
```

Comment:
```suggestion
import importlib
```

here we use importlib.metadata, so I think this import makes more sense